### ISSUE TITLE
Unable to side scroll for large markdown tables, resulting in tables half off page

### ISSUE THREAD

**Issue by wjkw1** [Requarks/wiki#4224]
### Discussed in https://github.com/Requarks/wiki/discussions/4021

<div type='discussions-op-text'>

<sup>Originally posted by **wjkw1** May 27, 2021</sup>
Hello,

While creating/editing a markdown page in Wiki.js, if we add a large table with many columns there is no scroll bar at the bottom. This results in table half off the screen.

**Example screenshot:**
![Screenshot here...](https://user-images.githubusercontent.com/30451128/119744740-c01cd980-bee0-11eb-98a5-d78ee1290927.png)

**Example markdown table text:**
```
|sa   | sa  | sa  | sa  | sa  |sa   |sa   |sa   |sa   |sa   |sa   | sa  | sa  | sa  | sa  |sa   |sa   |sa   |sa   |sa   |
|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|
|This is the only way to test it   | long wasy   | do i dail   | what the hceckes   | hwo ads   |   asdasdd|sad    |asdawdas   |asdwqeqwe   |saddsadsa   |This is the only way to test it   | long wasy   | do i dail   | what the hceckes   | hwo ads   |   asdasdd|sad    |asdawdas   |asdwqeqwe   |saddsadsa   |
|This is the only way to test it   | long wasy   | do i dail   | what the hceckes   | hwo ads   |   asdasdd|sad    |asdawdas   |asdwqeqwe   |saddsadsa   |This is the only way to test it   | long wasy   | do i dail   | what the hceckes   | hwo ads   |   asdasdd|sad    |asdawdas   |asdwqeqwe   |saddsadsa   |
|This is the only way to test it   | long wasy   | do i dail   | what the hceckes   | hwo ads   |   asdasdd|sad    |asdawdas   |asdwqeqwe   |saddsadsa   |This is the only way to test it   | long wasy   | do i dail   | what the hceckes   | hwo ads   |   asdasdd|sad    |asdawdas   |asdwqeqwe   |saddsadsa   |
```

Thanks
Western
</div>

**Host Info (please complete the following information):**
OS: Linux Ubuntu, Docker
Wiki.js version: [2.5]
Database engine: [Postgres]

**Comment by wjkw1** [comment]
@NGPixel - is something in the 3.0 pipeline?

**Comment by nasiralamreeki** [comment]
> @NGPixel - is something in the 3.0 pipeline?

My understanding is 2.x has some known UI issues that will be fixed in 3.0.

**Comment by NGPixel** [comment]
Please don't create duplicate tickets. This has already been reported.
